rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMService.java

```diff
@@ -330,4 +330,19 @@ String getNodeMBeansInfo(String sessionId, String nodeJmxUrl, String objectNames
     String executeNodeScript(String sessionId, String script, String engine, String nodeUrl)
             throws RestServerException, ServiceException;
 
+    /**
+     * Retrieves the thread dump of the Resource Manager
+     *
+     * @param sessionId current session
+     */
+    String getRMThreadDump(String sessionId) throws ServiceException, RestServerException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)